libs/sdk-model/src/execution/buckets/index.ts

```diff
@@ -137,8 +143,25 @@ export function newBucket(
                 } as one of the items for bucket ${localId}.` +
                     "Please make sure that you are not trying to put an array of items into a bucket that only accepts single item.",
             );
+        } else if (typeof i === "object") {
+            if (Object.keys(i).indexOf("Default") > -1) {
+                invariant(
+                    false,
+                    `${contentErrorMessage} It looks like you meant to use of the following display forms: ${Object.keys(
```

Comment:
Idea: Maybe providing also target metadata object identifier in the error message could be helpful? (same applies for fact aggregations)